### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if it d…

### PR DESCRIPTION
**Pull request by OVikhrov**
…isallowed by Policy
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+-- In case
+-- SDL receives OnSeekMediaClockTimer notification from HMI
+-- and this notification is valid and NOT allowed by Policies
+-- SDL must:
+-- log corresponding error internally
+-- ignore this OnSeekMediaClockTimer notification
+-- SDL must NOT:
+-- transfer OnSeekMediaClockTimer notification from HMI to mobile app
+
+--
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+-- b) OnSeekMediaClockTimer notification disallowed in preloaded file for default app for all HMI levels
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL in all HMI levels.
+--
+-- Expected result:
+-- a) SDL does not resend OnSeekMediaClockTimer notification to mobile app for all HMI levels.
+---------------------------------------------------------------------------------------------
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Policy_Disallowed_In_All_HMI_Levels.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**